src/components/skills-quiz/SearchCurrentJobCard.jsx

```diff
@@ -8,20 +8,22 @@ import { SkillsContext } from './SkillsContextProvider';
 import JobCardComponent from './JobCardComponent';
 import { SET_KEY_VALUE } from './data/constants';
 
-const SearchCurrentJobCard = ({ index }) => {
+function SearchCurrentJobCard({ index }) {
   const { refinements } = useContext(SearchContext);
   const { current_job: currentJob } = refinements;
   const [isLoading, setIsLoading] = useState(true);
   const { dispatch, state } = useContext(SkillsContext);
   const { currentJobRole } = state;
-  const jobToFetch = useMemo(() => {
-    const jobsArray = [];
-    if (currentJob?.length > 0) {
-      jobsArray.push(`name:${currentJob[0]}`);
-    }
-    return jobsArray;
-  },
-  [currentJob]);
+  const jobToFetch = useMemo(
+    () => {
+      const jobsArray = [];
+      if (currentJob?.length > 0) {
+        jobsArray.push(`name:${currentJob[0]}`);
+      }
+      return jobsArray;
+    },
+    [currentJob],
```

Comment:
[curious] Is `jobToFetch` re-computed on every render of `SearchCurrentJobCard`? I believe `currentJob` in the dependency array will have a new variable reference on each re-render, and thus cause `jobToFetch` to unnecessarily re-compute each re-render.

Reply:
Also nit: i think this variable might be intended to be named `jobsToFetch` since its a collection of multiple jobs?